src/app/components/Results/ResultsList.jsx

```diff
@@ -136,6 +137,11 @@ class ResultsList extends React.Component {
     const totalItems = items.length;
     const hasRequestTable = items.length === 1;
 
+    let bibUrl = `${appConfig.baseUrl}/bib/${bibId}`
+
+    if (this.props.searchKeywords) bibUrl += `?searchKeywords=${this.props.searchKeywords}`;
+    else if (Store.state.searchKeywords) bibUrl += `?searchKeywords=${Store.state.searchKeywords}`;
```

Comment:
Me be nitpicky about styling and readability:
```
if (...) {
  bibUrl += ...
} else if (...) {
  bibUrl += ...
}
```

Reply:
Agreed 👍